bin/docker-outline-dns

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Get docker ip
+docker_ip=$(ip -family inet -oneline addr show docker0|\
```

Comment:
Ooooh ok I think you mean "offline mode". Can you change that? Also, one PR is fine, as long as you make 2 commits :)